.gitlab-ci-check-docker-build.yml

```diff
@@ -155,6 +155,11 @@ build:docker-multiplatform:
       --provenance false
       --push
       ${DOCKER_DIR:-.}
+    - docker save ${GITLAB_REGISTRY_TAG} > ${IMAGE_ARTIFACT_FILENAME:-${CI_PROJECT_DIR}/image.tar}
```

Comment:
I'm not sure if this works without the `--load` argument to the `buildx build` command. Also, using `docker save` it will only save the image for the architecture executing the command since the local docker image store does support multi-platform images.